hazelcast/src/main/java/com/hazelcast/sql/SqlColumnType.java

```diff
@@ -119,4 +119,9 @@ public int getId() {
     public Class<?> getValueClass() {
         return valueClass;
     }
+
+    @Override
+    public String toString() {
```

Comment:
I'd rather not override `toString()`, but use a custom method, such as `sqlName()`. `toString()` is generic and used for any purpose, people tend to change it without much thinking. It's also hard to find callers of `toString()` in call hierarchy.